### PR TITLE
Fix Bug 1249928 - Firefox/Thunderbird beta release notes "download link" does not point to betas

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/firefox/templates/firefox/releases/release-notes.html [hunk 1]
```diff
@@ -253,7 +253,7 @@
       <div class="container">
         <div class="all-download">
           <p class="message">{{ _('Download the latest version of Firefox') }}</p>
-          <a rel="external" href="{{ firefox_url('desktop', 'all') }}">{{ _('All Firefox downloads') }}</a>
+          <a rel="external" href="{{ firefox_url('desktop', 'all', channel=channel_name|lower()) }}">{{ _('All Firefox downloads') }}</a>
```
Comment (jgmize) [inline bedrock/firefox/templates/firefox/releases/release-notes.html]: Why pipe the channel name through `lower()` when you're matching the `Beta` in the function itself?

**Comment by alexgibson** [comment]
Friendly ping @kyoshino - I think one small amend and this one is good to merge? 😄

File: bedrock/firefox/templates/firefox/releases/release-notes.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/releases/release-notes.html]: It looks like `lower()` is needed here, without it the page throws a `NoReverseMatch`

Comment (jgmize) [inline bedrock/firefox/templates/firefox/releases/release-notes.html]: thanks @alexgibson, now it makes sense.

**Comment by kyoshino** [comment]
Doh, this slipped under my radar. Sorry about it, and thanks for your review!

**Pull request merged**